docs/tutorials/03_shaders/viz_pbr_spheres.py

```diff
@@ -0,0 +1,136 @@
+"""
+===============================================================================
+Physically-Based Rendering (PBR) on spheres
+===============================================================================
+
+PBR engines aim to simulate properties of light when it interacts with objects
+in the scene in a physically plausible way. The interaction of light with an
+object depends on the material the object is made of. In computer graphics,
+materials are usually divided in 2 main categories based on their conductive
+properties: dielectrics and metals.
+
+This tutorial, illustrates how to model some material properties in FURY by
+using the PBR material.
+
+Let's start by importing the necessary modules:
+"""
+
+from fury import actor, material, window
+from fury.utils import (normals_from_actor, tangents_to_actor,
+                        tangents_from_direction_of_anisotropy)
+import numpy as np
+
+"""
+Now set up a new scene.
+"""
+
+scene = window.Scene()
+scene.background((.9, .9, .9))
+
+"""
+Let's define the parameters we are going to showcase in this tutorial.
+These subset of parameters have their values constrained in the 0 to 1 range.
+"""
+
+material_params = [
+    [[1, 1, 0], {'metallic': 0, 'roughness': 0}],
+    [(0, 0, 1), {'roughness': 0}],
+    [(1, 0, 1), {'anisotropy': 0, 'metallic': .25, 'roughness': .5}],
+    [(1, 0, 1), {'anisotropy_rotation': 0, 'anisotropy': 1, 'metallic': .25,
+                 'roughness': .5}],
+    [(0, 1, 1), {'coat_strength': 0, 'roughness': 0}],
+    [(0, 1, 1), {'coat_roughness': 0, 'coat_strength': 1, 'roughness': 0}]
+]
+
+"""
+Now we can start to add our actors to the scene and see how different values of 
+the parameters produce interesting effects.
+"""
+
+for i in range(6):
+    center = [[0, -5 * i, 0]]
+    for j in range(11):
+        center[0][0] = -25 + 5 * j
+        sphere = actor.sphere(center, material_params[i][0], radii=2, theta=32,
+                              phi=32)
+        normals = normals_from_actor(sphere)
+        tangents = tangents_from_direction_of_anisotropy(normals, (0, 1, .5))
+        tangents_to_actor(sphere, tangents)
+        keys = list(material_params[i][1])
+        material_params[i][1][keys[0]] = np.round(0.1 * j, decimals=1)
+        material.manifest_pbr(sphere, **material_params[i][1])
+        scene.add(sphere)
+
+"""
+For interpretability purposes we will add some labels to guide us through our 
+visualization.
+"""
+
+labels = ['Metallic', 'Roughness', 'Anisotropy', 'Anisotropy Rotation',
+          'Coat Strength', 'Coat Roughness']
+
+for i in range(6):
+    pos = [-40, -5 * i, 0]
+    label = actor.vector_text(labels[i], pos=pos, scale=(.8, .8, .8),
+                              color=(0, 0, 0))
+    scene.add(label)
+
+for j in range(11):
```

Comment:
what is this `11`?

Reply:
The 11 spheres that we are rendering with different material properties. I will add a variable name to improve readability. Thanks for catching that.

Reply:
Changed.